sdk-distributors/sdk/disk-partitions/index.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: docs
+title: Disk Layout
+category: sdk_distributors
+sub_category: sdk
+weight: 9
+---
+
+# CoreOS Disk Layout
+
```

Comment:
"The operating system has 9 different disk partitions, utilizing a subset of those to make each update safe and enable a roll-back to a previous version if anything goes wrong."